### PR TITLE
First part of fixing the docs. 

### PR DESCRIPTION
**Pull request by MihaiTabara**
I'll follow-up next week after we have some more answers to some of the TODs to streamline the process.

### CODE REVIEW

File: docs/mergeduty/howto.md [hunk 1]
```diff
@@ -39,7 +37,8 @@ How are those repositories kept in sync? That's `MergeDuty` and is part of the `
 
 Historical context of this procedure:
 
-Originally, the `m-c` -> `m-b` was done a week after `m-b` -> `m-r`. Starting at `Firefox 57`, Release Management wanted to ship DevEdition `b1` week before the planned mozilla-beta merge day. This meant Releng had to merge both repos at the same time.
+~Originally, the `m-c` -> `m-b` was done a week after `m-b` -> `m-r`. Starting at `Firefox 57`, Release Management wanted to ship DevEdition `b1` week before the planned mozilla-beta merge day. This meant Releng had to merge both repos at the same time.~
+**TODO** - to rewrite the process here once 71 is out of the door.
```
Comment (lundjordan) [inline docs/mergeduty/howto.md]: LOL we need a "originally, originally" now :)

File: docs/mergeduty/howto.md [hunk 2]
```diff
@@ -142,10 +141,10 @@ screen
 source /home/buildduty/mergeday/bin/activate
 mkdir merge_day_${RELEASE_VERSION_FOR_CYCLE}
 cd merge_day_${RELEASE_VERSION_FOR_CYCLE}
-wget -O mozharness.tar.bz2 https://hg.mozilla.org/mozilla-central/archive/tip.tar.bz2/testing/mozharness/
-tar --strip-components=2 -jvxf mozharness.tar.bz2
-wget -O mozbase.tar.bz2 https://hg.mozilla.org/mozilla-central/archive/tip.tar.bz2/testing/mozbase/
-tar --strip-components=2 -jvxf mozbase.tar.bz2
+wget -O mozharness.zip https://hg.mozilla.org/mozilla-central/archive/tip.zip/testing/mozharness/
```
Comment (lundjordan) [inline docs/mergeduty/howto.md]: ooo. hmm.

File: docs/mergeduty/howto.md [hunk 3]
```diff
@@ -198,6 +197,7 @@ Diff should be similar to [this one](https://hg.mozilla.org/releases/mozilla-esr
 Make sure the bug that tracks the migration has no blocking items.
 
 ### Land whatsnewpage list of locales
+**TODO** - this needs to change, as the process no longer assumes this, but apply them; the l10n drivers provide the final list of locales to receive the WNP on the Tuesday prior to the ship date.
```
Comment (lundjordan) [inline docs/mergeduty/howto.md]: we always use the friday before first merge day as the due date for locales. which tues is this for now?

Comment (MihaiTabara) [reply docs/mergeduty/howto.md]: Let's talk more at the post-mortem, I add more issues there. But Tuesday as in tomorrow.

**Pull request merged**